### ISSUE TITLE
Quick-add time entries

### ISSUE THREAD

**Issue by muety** [te-online/timemanager#68]
First of all: great project, I'm curious to see where it goes! I'm currently using Toggl to track my time day-to-day, but have been looking for a self-hosted, open-source alternative for a while. Since I'm already managing tasks, etc. in Nextcloud, this would be a good place for my time tracking, too. Unfortunately, I found that most tools, including the awesome [kevinpapst/kimai2](https://github.com/kevinpapst/kimai2) are just way too sophisticated for my needs. Instead, I'm looking for a time tracker that focused on simplicity and good user experience.

**TL;DR**

What I'd really like to see with Nextcloud TimeManager is the ability to quick-add time entries, just like with Toggl. This actually includes two "features":

1. Ability to **start- and stop** time entries, instead of specifying a duration, i.e. have kind of a "live tracker"
2. **Hit and go** mode, where you only type a description and press start, instead of first having to create a client, project and task

Are these something you would consider for future releases?

**Comment by te-online** [comment]
Hey @muety Thanks for your kind words 🙂 

_Don't tell anyone, but I'd actually like it to be able to compete with toggl, too 🤭_ 
But as you see and describe, we're not there right now and likely won't be for a long time, since it's only me working on this app so far and I have my requirements covered, a few UX-annoyances aside. That doesn't mean I don't _want_ to make improvements. It just means that help is more than appreciated if you need something specific, since I can't carve out time for anything more right now, hope that makes sense 🙂 

Regarding your feature requests, I think (1) is covered in https://github.com/te-online/timemanager/issues/11 and https://github.com/te-online/timemanager/issues/28 and (2) is something I have thought about myself a lot...

**Comment by muety** [comment]
Thanks for your detailed answer! I'll close this as it's pretty much a duplicate of #11 and curiously wait for that one to be implemented then. Maybe I'll even find some time to contribute sooner or later.
